.appveyor.yml

```diff
@@ -54,9 +54,7 @@ before_test:
   - msbuild fabtests.sln
 
 test_script:
-  - set PATH=%CD%\x64\%CONFIGURATION%;%PATH%
-  - set FI_PROVIDER=sockets
-  - scripts\runfabtests.cmd
+  - scripts\runfabtests.cmd -v -v -v -p %CD%\x64\%CONFIGURATION% -t all sockets
```

Comment:
We may need to keep this as quick or short tests for CI purposes.  We can also consider changing the script to limit the number of iterations to 5, even for the long tests.